js/everything.js

```diff
@@ -206,11 +212,55 @@ var ZenPlayer = {
                     seconds: plyrPlayer.plyr.embed.getDuration(),
                     youtubeID: plyrPlayer.plyr.embed.getVideoData().video_id
                 });
-
```

Comment:
It's not complaining on `master`, there's probably some indents on the line, remove those and it should be good.